cloud/aws/policies.tf

```diff
@@ -59,10 +59,6 @@ resource "aws_iam_policy" "sfx_policy" {
             "lambda:GetAlias",
             "lambda:ListFunctions",
             "lambda:ListTags",
-            "logs:DeleteSubscriptionFilter",
-            "logs:DescribeLogGroups",
```

Comment:
Are you confident these removed permissions are not needed the basic AWS integration?

Reply:
Not really, I rely on https://registry.terraform.io/providers/splunk-terraform/signalfx/latest/docs/resources/aws_integration#enable_logs_sync description but will ask to support

Reply:
Hello @pdecat now we are sure and the official doc has been updated: https://docs.splunk.com/Observability/gdi/get-data-in/connect/aws/aws-apiconfig.html#review-aws-iam-policy-permissions-by-observability-cloud-feature

So the changes was ok according to it and in the current state this modules uses:
- a common policy which is a merge of `Tag and property collection` and `metric collection`
- a policy dedicated to logs including only the required permission in addition to this common policy
- a policy dedicated to metric streams

Everything seem good for except if we prefer to split the common policy between 2 policies: metric collection vs tags sync but it seems useless for me.